packages/starlight/components/Search.astro

```diff
@@ -72,13 +72,15 @@ const pagefindTranslations = {
 
 			const openModal = (event?: MouseEvent) => {
 				dialog.showModal();
+				document.body.toggleAttribute('data-search-modal-open', true);
```

Comment:
I added the force option here, just to *never* get in a situation where it's the wrong way around for any weird reason.

This could work, too.

```suggestion
				document.body.toggleAttribute('data-search-modal-open');
```